src/types/src/defaults.rs

```diff
@@ -29,6 +29,7 @@ pub const SPU_LOG_SIZE: &str = "1Gi";
 pub const SPU_LOG_INDEX_MAX_BYTES: u32 = 10485760;
```

Comment:
Can you move new constant to storage crates?  We are going to deprecrate this crate in future.

Reply:
yes this is a simple change